quarkus-workshop-super-heroes/super-heroes/rest-fights/src/main/resources/application.properties

```diff
@@ -11,6 +11,10 @@ io.quarkus.workshop.superheroes.fight.client.HeroProxy/mp-rest/url=http://localh
 io.quarkus.workshop.superheroes.fight.client.HeroProxy/mp-rest/scope=jakarta.inject.Singleton
 io.quarkus.workshop.superheroes.fight.client.VillainProxy/mp-rest/url=http://localhost:8084
 io.quarkus.workshop.superheroes.fight.client.VillainProxy/mp-rest/scope=jakarta.inject.Singleton
+# tag::adocNarrate[]
+io.quarkus.workshop.superheroes.fight.client.NarrationProxy/mp-rest/url=http://localhost:8086
```

Comment:
Not part of this PR, but we should probably use the config form that makes this a lot simpler. I can do that in a follow up for all the REST Clients involved